.github/workflows/update-changelog.yaml

```diff
@@ -17,16 +19,17 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: main
+          persist-credentials: false
```

Comment:
I also wonder if the `stefanzweifel/git-auto-commit-action` is relying on having the original git credentials

Reply:
Based on [this](https://github.com/stefanzweifel/git-auto-commit-action?tab=readme-ov-file#action-does-not-push-commit-to-repository-authentication-issue) it seems like that action relies on the permissions/credentials given to the `checkout` action, so I think the answer is yes.